orchestrator/dags/general_data_import.py

```diff
@@ -220,24 +217,23 @@ def extract_routes_lines(ti) -> None:
     general_file_name = ti.xcom_pull(key='general_file_name')
     routes_output_file = "routes_file.txt"
     with open(f"{out_dir}{general_file_name}", "rt", encoding="utf-8") as file:
-        f = open(routes_output_file, "w", encoding='utf-8')
-        previous_line = ""
-        for line in file:
-            if '*TR' in line or '*LW' in line:
-                try:
-                    f.write(previous_line)
-                except StopIteration:
-                    # there is no lines left
-                    break
-                while '#LW' not in line:
+        with open(routes_output_file, "w", encoding='utf-8') as f:
+            previous_line = ""
+            for line in file:
+                if '*TR' in line or '*LW' in line:
                     try:
-                        f.write(line)
-                        line = next(file)
+                        f.write(previous_line)
```

Comment:
Is this `try/except` block neccessary? Probably refering to `previous_line` cannot raise `StopIteration`.